analyses/molecular-subtyping-integrate/input/tcga_pathology_diagnosis_to_histology_mapping.tsv

```diff
@@ -0,0 +1,34 @@
+pathology_diagnosis	broad_histology	short_histology	cancer_group
```

Comment:
hmm can you take broad/short histology out of this since I had changed one or more short histologies upstream ?

Reply:
@jharenza, I had asked the following on slack and you said ok. 
https://opentargetspediatrics.slack.com/archives/C021ZQ0C7KM/p1654783501445249

Based on the latest `histologies-base .tsv` in the D3b `v11-qc` PR, the following snippet of the code will integrate for non-PBTA 
https://github.com/ewafula/OpenPedCan-analysis/blob/5566f0e4041f2fcc4a5129a665ac01548d2385a9/analyses/molecular-subtyping-integrate/01-integrate-subtyping.Rmd#L168-L200

Please look at the results/histologies.tsv. 

I set broad/short histology in the latest `v11 histologies-base .tsv`  to `NA` and then revert back to how I had previously updated the code to integrate `broad_histology`, `short_histology,` and `cancer_group` for non-PBTA, including using a mapping file for TCGA.